cpp/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.apache.tsfile</groupId>
         <artifactId>tsfile-parent</artifactId>
-        <version>1.0.1-SNAPSHOT</version>
+        <version>1.0.1-tsfilev4-240801-SNAPSHOT</version>
```

Comment:
Should we keep the version 1.0.1-SNAPSHOT? We can change it later after the PR merged.

Reply:
This PR is just to trigger the CI, and I will not merge it into 'develop'.
Actually, I will merge it back to tsfile_v4.

Reply:
How about we update it to 1.2.0-SNAPSHOT now?